apps/api-reference/src/components/EvmProvider/index.tsx

```diff
@@ -19,9 +19,11 @@ const CHAINS = NETWORK_IDS.map((id) =>
 
 const TRANSPORTS = Object.fromEntries(
   CHAINS.map((chain) => {
-    const url = getRpcUrl(chain.id);
-    if (url) {
-      return [chain.id, http(url)];
+    const rpcUrl = getRpcUrl(chain.id);
+    if (rpcUrl) {
+      return [chain.id, http(rpcUrl)];
+    } else if (chain.rpcUrls.default.http[0]) {
+      return [chain.id, http(chain.rpcUrls.default.http[0])];
```

Comment:
This logic should live in `getRpcUrl` -- it doesn't make sense that we have a function that should get the rpc URL which only sometimes returns the RPC url and other times there is still a valid URL but the consumer needs to know about different logic to retrieve it.